pandas/io/formats/style.py

```diff
@@ -1454,9 +1455,8 @@ def set_properties(self, subset=None, **kwargs) -> Styler:
         >>> df.style.set_properties(color="white", align="right")
         >>> df.style.set_properties(**{'background-color': 'yellow'})
         """
-        values = ";".join(f"{p}: {v}" for p, v in kwargs.items())
-        f = lambda x: values
-        return self.applymap(f, subset=subset)
+        values = "".join(f"{p}: {v};" for p, v in kwargs.items())
+        return self.applymap(lambda x: values, subset=subset)
```

Comment:
this was a pep8 cleanup (not allowing definitions by lambda) and cleaning the `values` string to end in ";"